test/test_integration_list.py

```diff
@@ -30,24 +29,23 @@ def setup_cli_params(self, env_vars, *params):
 
     def setUp(self):
         self.setup_cli_params({})
-        # Need to configure the CLI and login to perform further commands
+        # Used to configure the CLI and login to run tests
         Utils.setup_cli(self)
-        return self.invoke_cli(self.cli_auth_params,
-                       ['policy', 'replace', 'root', self.environment.path_provider.get_policy_path("list")])
+        self.invoke_cli(self.cli_auth_params,
+                       ['policy', 'replace', '-b', 'root', '-f', self.environment.path_provider.get_policy_path("list")])
 
     @integration_test
```

Comment:
You can add here the same as you did before
 # *************** TESTS ***************

Reply:
added